app/javascript/src/components/Navigation/components/GlobalActivitiesModal.jsx

```diff
@@ -42,51 +42,108 @@ const StyledBottom = styled(Button)`
 
 const StyledModalBody = styled(Modal.Body)`
   background-color: ${COLOR_CONCRETE};
-  color: ${COLOR_MINE_SHAFT};;
+  color: ${COLOR_MINE_SHAFT};
 `;
 
-class GlobalActivitiesModal extends Component {
-  constructor(props) {
+type Props = {
+  more: boolean,
+  showModal: boolean,
+  onCloseModal: Function,
+  activites: Array<Activity>
```

Comment:
Are you still using `more` and `activites` props? Didn't you moved them to State?